docs/testing.md

```diff
@@ -524,6 +524,67 @@ class PostCountBadgeTest extends TestCase
 }
 ```
 
+Sometimes it may come in handy to assert that an event was dispatched with one or more parameters. Let's have a look at a component called `PostsTable` that dispatches an event called`banner-message` with parameters `message` and `style`:
```

Comment:
I think you're missing a space between "called" and "`banner-message`"

Reply:
You are right. Fixed it! Thanks